frontend/src/Pages/AdminManagement.tsx

```diff
@@ -168,107 +168,102 @@ export default function AdminManagement() {
                         data-tip="Add Administrator"
                     >
                         <button
-                            className="btn btn-primary btn-sm"
+                            className="btn btn-primary btn-sm text-white" // Makes the text white
                             onClick={() => addUserModal.current?.showModal()}
                         >
-                            <UserPlusIcon className="h-4" />
+                            <PlusIcon className="h-4 border border-white rounded-full p-1 mr-2" />{' '}
+                            {/* Circular border around PlusIcon */}
+                            Add Admin
                         </button>
                     </div>
                 </div>
-                <table className="table">
-                    <thead>
-                        <tr className="border-gray-600">
-                            <th className="flex">
-                                <span>Name</span>
-                            </th>
-                            <th>Username</th>
-                            <th>Last Updated</th>
-                            <th>Actions</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {!isLoading &&
-                            !error &&
-                            userData.map((user: User) => {
-                                const updatedAt = new Date(user.updated_at);
-                                return (
-                                    <tr
-                                        key={user.id}
-                                        className="border-gray-600"
-                                    >
-                                        <td>
-                                            {user.name_first} {user.name_last}
-                                        </td>
-                                        <td>{user.username}</td>
-                                        <td>
+                <div className="w-full space-y-4">
+                    <div className="relative">
+                        <div className="grid grid-cols-4 px-6 mb-2 text-sm font-medium text-gray-500">
```

Comment:
I received feedback to this on a similar PR but we should try not to use classes like `gray-500` or `gray-200`. There are some values found within `tailwind.config.js` like `grey-1` and `grey-2`, etc. that we should use. 

The reason we should use the ones found in `tailwind.config.js` is because they are set up to work with light mode and dark mode where as `gray-500` won't change.